docs/superpowers/plans/2026-03-30-pr-automation.md

```
@@ -0,0 +1,809 @@
+| `fix/` | Bug fix |
+| `refactor/` | Code restructure, no behaviour change |
+| `ci/` | CI/CD changes |
+| `chore/` | Tooling, deps, config |
+| `deps/` | Dependency bumps |
+
+## Rules
+
+- **Never push directly to `main`** — always use a PR
+- **Squash merge only** — never regular merge or rebase-merge
+- **Never use `--no-verify`** — all hooks must pass
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)